src/vma/dev/cq_mgr.cpp

```diff
@@ -1600,3 +1599,356 @@ void cq_mgr::mlx5_init_cq()
 	m_mlx5_cqes = (volatile struct mlx5_cqe64 (*)[])(uintptr_t)m_mlx5_cq->active_buf->buf;
 }
 #endif // DEFINED_VMAPOLL
+
+#ifdef HAVE_INFINIBAND_MLX5_HW_H
+cq_mgr_mlx5::cq_mgr_mlx5(ring_simple* p_ring, ib_ctx_handler* p_ib_ctx_handler, uint32_t cq_size, struct ibv_comp_channel* p_comp_event_channel, bool is_rx):
+	cq_mgr(p_ring, p_ib_ctx_handler, cq_size, p_comp_event_channel, is_rx)
+	, m_cq_size(cq_size)
```

Comment:
You prohibit using your implementation (that is, no use of `cq_mgr_mlx5`) when `DEFINED_VMAPOLL`. So these parameters are used **mutual elusively**. I don't see why have two members - both with the same semantics - whose usage never overlaps.
I don't mind which of them it is (I agree with your considerations to use similar names to those found in _libibverbs_ / _libmlx5_).
If there's a compelling argument against this - please explain.